### DISCUSSION TITLE
Feedback on migration of a website.

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#37623] [Show and tell]
https://nextprojectb.vercel.app/

^^ Above is an old website that I created using just html and bootstrap.. the site was later converted into REACT, and now I was looking to do the the migration into Next.js...      IcyJoseph  was very helpful to the Next.js migration and I would still be working on it without his help.

I give the migration 8 out of 10..

There are few things to learn in Next such handling of CSS, pictures, and coding without having the front end available right away... not too bad overall...

A portion of the pain was dealing the the CSS, but this was mostly to a mistake in the import sequence of the bootstrap and global css in the app....

The very biggest pain was trying to get data from MongoDB-Atlas.  If I was a person heavily invested in MongoDB-Atlas I would have given Next a not so great rating, but  MongoDB-Atlas is not on my favorite side to begin with....  whatever that is "different" between REACT and Next to get data from MongoDB-Atlas maybe should be presented more clearly at the database company?  knowing javascript or REACT doesn't mean you will be able to crack the code for the database connection without further details..

Anyway.... not too terrible.....  I will be adding to this site more stuff, and will be building other sites in Next.... at the moment Next is the best tool I see for making apps that are SEO friendly.

**Comment by HUMBLEGROUND** [comment]
I'll take a look at the site you shared.
